src/components/manage/Blocks/StatisticBlock/Edit.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SidebarPortal } from '@plone/volto/components';
+import InlineForm from '@plone/volto/components/manage/Form/InlineForm';
+import View from './View';
+import schema from './schema';
+
+const Edit = (props) => {
```

Comment:
@razvanMiu I created a dedicated Volto add-on for this block. See [volto-statistic-block](https://github.com/eea/volto-statistic-block)